### DISCUSSION TITLE
Use a non-default schema in SQL replication

### DISCUSSION THREAD

**Discussion by krivulcik** [ravendb/ravendb#12837] [Q&A] [answered]
Let's say that we have an SQL table in a non-default schema.

CREATE TABLE [myschema].[Table] ( ... ).

How do I specify such a table in the transformation script?

The table is specified with TableName: "[myschema].[Table]", I can verify that in the request payload.

However, I'm unable to use something like loadTo[myschema].[Table], as the javascript parser doesn't accept this: Expected an identifier and instead saw '['.

Dropping the brackets doesn't help: Object has no method 'Table'.

Using only the table name in the script (loadToTable) doesn't work: Table 'Table' was not defined in the configuration of SQL ETL task

From the source code, it seems that using loadTo("myschema.Table", object) should be usable, but no: Invalid ETL configuration for '...'. Reason: No `loadTo<TableName>()` method call found in 'Script_1' script.

Is it possible to use a SQL schema with SQL replication? How?

**Comment by arekpalinski** [comment]
Hi @krivulcik 

This is supposed to work this way: `loadTo('myschema.Table', ... );`. Although there is a problem with property recognizing `myschema.Table` as a table name. I will be fixing it

**Reply by arekpalinski** [reply]
Opened issue for that: https://issues.hibernatingrhinos.com/issue/RavenDB-17327

**Comment by arekpalinski** [comment, marked as answer]
Fixed in https://github.com/ravendb/ravendb/pull/12840
